maoni/src/main/java/org/rm3l/maoni/utils/ContextUtils.java

```diff
@@ -53,8 +55,17 @@ public static Object getBuildConfigValue(@NonNull final Context context,
                     .getField(fieldName)
                     .get(null);
         } catch (final Exception e) {
-            //No worries
-            e.printStackTrace();
```

Comment:
Removed `printStackTrace`  since it leads to some unnecessary noise for apps that integrate with `Maoni`